analytics/js_tracking_api.rst

```diff
@@ -859,3 +859,32 @@ To disable tracking user anonymously (after visitor gave consent) use ``deanonym
 
     _paq.push(["deanonymizeUser"]);
 
+
+
+Gethering navigation timing page performance metrics
+``````````````````````````
+To set up page performance metrics gathering use the ``setTimingDataSamplingOnPageLoad`` function::
+
+    _paq.push(["setTimingDataSamplingOnPageLoad", updateTimingDataOnPageLoadSampling]);
```

Comment:
Please add information when a user should run this `_paq.push`, before or after `trackPageView` etc.